File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer> seen = new HashSet<>();
        for (int num:nums){
            seen.add(num);
        }
        int res = 0;
        for (int num:nums){
            if (seen.contains(num-1)) continue;

            int curLen = 0;
            int temp = num;
            while (seen.contains(temp)){
                curLen++;
                res = Math.max(res, curLen);
                temp++;
            }
        }

        return res;
    }
    
}

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        n1 = n2 = -1
        c1 = c2 = 0

        for num in nums:
            if num == n1:
                c1 += 1
            elif num == n2:
                c2 += 1
            elif c1 == 0:
                n1 = num
            elif c2 == 0:
                n2 = num
            else:
                c1 -= 1
                c2 -= 1
        
        c1 = c2 = 0
        for num in nums:
            if num == n1:
                c1 += 1
            if num == n2:
                c2 += 1
        
        res = []
        if c1 // 3 > 0:
            res.append(n1)
        if c2 // 3 > 0:
            res.append(n2)

        return res